app/code/Magento/Newsletter/Test/Mftf/Test/CustomTestNewsletter.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="CustomTestNewsletter">
+        <annotations>
+            <features value="Newsletter"/>
+            <group value="Newsletter"/>
+            <title value="Newsletter subscription when user is registered without confirmation"/>
+            <description value="Newsletter confirmation email can't be sent again for customer created from admin panel
+            when registration confirmation is disabled and newsletter confirmation is enabled"/>
+        </annotations>
+        <before>
+            <actionGroup ref="EnableConfirmNewsletterDisableRegistrationConfirmActionGroup" stepKey="enableConfirmation"/>
+        </before>
+        <after>
+            <actionGroup ref="DisableConfirmNewsletterEnableRegistrationConfirmActionGroup" stepKey="disableConfirmation"/>
+            <actionGroup ref="logout" stepKey="adminLogout"/>
```

Comment:
You should add login as admin in **before** section
Did you run this test? It should fail without admin login